bindings/matrix-sdk-ffi/src/event.rs

```diff
@@ -0,0 +1,99 @@
+use anyhow::{bail, Context};
+use ruma::events::{AnySyncMessageLikeEvent, AnySyncTimelineEvent};
+
+use crate::{ClientError, MessageType};
+
+pub struct TimelineEvent(pub(crate) AnySyncTimelineEvent);
+
+#[uniffi::export]
```

Comment:
@jplatte Why the attribute here gives me an error?
It says:

required for `std::sync::Arc<event::TimelineEvent>` to implement `uniffi::FfiConverter<UniFfiTag>`

However I have defined in the .dul an empty interface implementation just as for RoomMember or TimelineItem

Reply:
The empty interface definition inside the UDL does the right thing, but only if `TimelineEvent` is visible in the place the UDL scaffolding is included (the crate root). You should add `event::*` to the import / re-export at line 54 of `lib.rs`.